src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        int randomNum = new Random().nextInt(100);
+        Ball nextBall = new Ball(randomNum, ColorSupplier.getRandomColor());
+        return nextBall;
```

Comment:
```suggestion
        return new Ball(randomNum, ColorSupplier.getRandomColor());
```
можна одразу повертати